modules/library.js

```diff
@@ -0,0 +1,20 @@
+export default class Library {
+  constructor() {
+    this.books = [];
+  }
+
+  addBook(book) {
+    this.books.push(book);
+  }
+
+  removeBook(book) {
+    const index = this.books.indexOf(book);
+    if (index !== -1) {
+      this.books.splice(index, 1);
+    }
+  }
+
+  getBooks() {
+    return this.books;
+  }
```

Comment:
- [ ] Please change these functions to arrow functions to follow the ES6 syntax as requested in the project requirements.